### DISCUSSION TITLE
Missing ordering of tooltip values in Time series visualization

### DISCUSSION THREAD

**Discussion by michal-sankot** [grafana/grafana#38305] [Dashboards, Panels & Visualizations]
**What happened**: It's not possible to order values in Tooltip in time series visualisation

**What you expected to happen**: Have a possibility to order values in Tooltip in time series visualisation

**How to reproduce it (as minimally and precisely as possible)**: Create a Time series panel

**Anything else we need to know?**: It was possible to set it in now deprecated Graph panel where we were using it. As we transitioned to Time series, we miss this possibility as tooltip contains a lot of values and we need to see the highest one on top. It thus presents a usability issue for us.

**Environment**:
- Grafana version: 8.0.2
- Data source type & version: Prometheus
- OS Grafana is installed on: Linux
- User OS & Browser: Chrome
- Grafana plugins: n/a
- Others: n/a

**Comment by jan25** [comment]
This seems interesting, could i try to fix the issue?

Maybe we could provide decreasing order as default. Or provide a way to choose 'Increasing' and 'Decreasing' when user selects 'Multi'(All) mode for tooltip. Any thoughts which might be best?

Also, tooltip isn't updated when editing a Timeseries panel's Tooltip settings, maybe state changes in frontend are buggy for Timeseries. But after 'Apply' it works fine.

<img width="1106" alt="Screenshot 2021-06-23 at 23 54 14" src="https://user-images.githubusercontent.com/12004246/123173077-53b0de00-d47e-11eb-8192-a68ba15620d2.png">

**Comment by towolf** [comment]
> Maybe we could provide decreasing order as default

I'm all for this, because I ALWAYS switch to decreasing. It's just the most useful setting to explore the data quickly.

But I can imagine that this is a contentious question.

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by racerb73** [comment]
This issue makes it very difficult to view the tooltip if there are lots of values in it.  It presents a usability issue for us as well.

**Comment by dprokop** [comment]
This is already in place in Grafana 8.4 and later improved in Grafana 8.5:
- https://github.com/grafana/grafana/pull/43615
- https://github.com/grafana/grafana/pull/49084
